src/editor/event-router.ts

```diff
@@ -135,7 +135,10 @@ export class EventRouter {
                         toRight: true,
                     });
                 } else if (this.module.validator.isTknEmpty(context)) {
-                    if (this.module.validator.isAugmentedAssignmentModifierStatement(context)) {
+                    if (
+                        this.module.validator.isAugmentedAssignmentModifierStatement(context) ||
+                        this.module.validator.isMethodCallModifierStatement(context)
```

Comment:
I didn't understand this. could you please add a comment or just explain things here? Thanks @just6660